### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jbellmann**
with instructions on how to use it in a K8s environment and most common issues people make during migration from STUPS to K8s.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,47 +2,99 @@
...
             - tokenId: secondService
-              scopes: singleScope:all
-
-### Usage
+              scopes: all
+```
+
+### Migration from Zalandos STUPS env to Zalandos K8s env
+
+Most common issue users do is not mounting the the credentials. Make sure you have something like this
```
Comment (lotharschulz) [inline README.md]: > Most common issue users do is not mounting the the credentials. Make sure you have something like this in your `deployment.yaml`

I propose: 

A common issue is not mounting the the credentials. Please use the example below as a guide line.

File: README.md [hunk 2]
```diff
@@ -2,47 +2,99 @@
...
+          - name: "{{ APPLICATION }}-credentials"
+            mountPath: /meta/credentials
+            readOnly: true
+      volumes:
+        - name: "{{ APPLICATION }}-credentials"
+          secret:
+            secretName: "{{ APPLICATION }}-credentials"
+```
+
+Another issue would be that identifiers/names for tokens have to be same as before in your `credentials.yaml`:
```
Comment (lotharschulz) [inline README.md]: > Another issue would be that identifiers/names for tokens have to be same as before in your `credentials.yaml`

Please also make sure that token identifiers/names must equal the respective items in `credentials.yaml`:

**Comment by jbellmann** [comment]
👍

File: README.md [hunk 3]
```diff
@@ -2,47 +2,98 @@
...
             - tokenId: secondService
-              scopes: singleScope:all
-
-### Usage
+              scopes: all
+```
+
+### Migration from Zalandos STUPS env to Zalandos K8s env
+
+A common issue is not mounting the the credentials. Please use the example below as a guide line.
```
Comment (lotharschulz) [inline README.md]: > A common issue is not mounting the the credentials.

Please make sure the credentials are mounted as shown in the example below.

**Comment by lotharschulz** [comment]
👍

**Comment by jbellmann** [comment]
👍

**Pull request merged**